ts-framework/create-function/gram-template-mcp/README.md

```diff
@@ -0,0 +1,43 @@
+# Gram Function MCP Template
+
+This template allows you to use the official [MCP TypeScript SDK][mcp-ts] to
+build and deploy [Gram Functions](https://getgram.ai).
+
+[mcp-ts]: https://github.com/modelcontextprotocol/typescript-sdk
+
+Gram Functions are tools and resources for LLMs and MCP servers that can do
+arbitrary tasks such as fetching data from APIs, performing calculations, or
+interacting with hosted databases.
```

Comment:
```suggestion
Use Gram Functions to build tools and resources for MCP servers. They can do
arbitrary tasks such as fetching data from APIs, performing calculations, or
interacting with hosted databases.
```

Thoughts? Just thinking that gram functions themselves aren't tools, but instead they _make_ tools.